### DISCUSSION TITLE
Edge margin style

### DISCUSSION THREAD

**Discussion by Brunogomes97** [xyflow/xyflow#2035] [Q&A]
Is there any way to add a margin style to an edge? Similar to this case:

![marginEdge](https://user-images.githubusercontent.com/36930139/161042458-700862e0-7d3d-4273-9e50-7c26a9c51e08.png)

**Comment by ghost** [comment]
A quick solution can be to give your handles a negative `top` and `right`

For example: 
```<Handle type="target" position={Position.Top} style={{ top: '-8px' }} />```

Another option is creating a custom edge.
